README.md

```diff
@@ -0,0 +1,73 @@
+[![image](https://img.shields.io/badge/License-BSD%203--Clause-blue.svg)](https://opensource.org/licenses/BSD-3-Clause)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+
+
+
+# atoMEC: Average-Atom Code for Matter under Extreme Conditions
+atoMEC is a python-based average-atom code for simulations of high energy density phenomena such as in warm dense matter. 
+It is designed as an open-source and modular python package. 
+
+atoMEC uses Kohn-Sham density functional theory, in combination with an average-atom approximation,
+to solve the electronic structure problem for single-element materials at finite temperature.
+
+More information on the average-atom methodology and Kohn-Sham density functional theory can be found (for example) in this [preprint](https://arxiv.org/abs/2103.09928) and references therein.
+
+This repository is structured as follows:
+```
+├── atoMEC : source code
+├── docs : sphinx documentation
+├── examples : useful examples to get you started with the package
+└── tests : test scripts used during development, will hold tests for CI in the future
+```
+
+
+
+## Installation
+First, clone the atoMEC repository and ``cd`` into the main directory.
+
+* Recommended : using [pipenv](https://pypi.org/project/pipenv/)
+
+  This route is recommended because `pipenv` automatically creates a virtual environment and manages dependencies.
+  
+  1. First, install `pipenv` if it is not already installed, for example via `pip install pipenv` (or see [pipenv](https://pypi.org/project/pipenv/) for    installation instructions)
+  2. Next, install `atoMEC` and its dependencies with `pipenv install`
```

Comment:
With `pipenv install` you don't actually install `atoMEC`, you "only" install atoMEC's dependencies.
```suggestion
  2. Next, install `atoMEC`'s dependencies with `pipenv install`
```